test/client_combined_test.go

```diff
@@ -194,7 +194,7 @@ func TestCombinedWithChangingObjectTypes(t *testing.T) {
 
 	rebuild(tc, c, 1, i(2), tmpDir, nil, expectedResponse{
 		version: 2,
-		count:   3,
+		count:   5,
```

Comment:
Do you know why all of the file counts changed?

Reply:
I think but I am not sure that it's the full summary emitting more removes than were detected before. When a file becomes a folder for example, we get a `{path: "a", op: "REMOVE"}` in addition to a `{path: "a/", op: "ADD"}` that I don't think we got before without doing the full diff. I didn't log them out though, I can prove that's the case.